README.rst

```diff
@@ -35,7 +35,7 @@ Horovod
 |
 
 Horovod is a distributed training framework for TensorFlow, Keras, PyTorch, and MXNet. The goal of Horovod is to make
-distributed Deep Learning fast and easy to use.
+distributed Deep Learning fast and easy to use. Full documentation can be found at https://horovod.readthedocs.io/en/latest.
```

Comment:
Instead of putting this here, can we move this below the "Contents" and say something like: 

"Full documentation and an API reference can be found `here <https://horovod.readthedocs.io/en/latest>`_."

Reply:
I agree with this suggestion. "See the full documentation and an API reference at https://horovod.readthedocs.io/en/latest."